schemas/context/person.schema.json

```diff
@@ -67,5 +67,5 @@
       "$ref": "https://ns.adobe.com/xdm/common/auditable#/definitions/auditlog"
     }
   ],
-  "meta:status": "experimental"
+  "meta:status": "stabilizing"
```

Comment:
#237 might change `xdm:name` to an array. 👎

Reply:
We don't have specific plans to change this, and Person is needed for the milestone, so I feel we need to take it forward.  We can decide whether we want to address #237 before 1.0 or not.

Reply:
👎 in addition to the name issue, the birthdate is handled incorrectly.  It should be a single date field in ISO 8601 format.